packages/api/config/tables.yml

```diff
@@ -2,6 +2,19 @@ CollectionsTable:
   read: 5
   write: 1
   pointInTime: '{{enablePointInTime}}'
+  enableAutoScaling: '{{tables.collections.enableAutoScaling}}'
+  autoScaling:
+    roleArn: '{{iams.scalingRoleArn}}'
+    readCapacityUnits:
+      min: '{{tables.collections.readCapacityUnits.min}}'
```

Comment:
Wherever this ends up (again, I think it should be condensed into a single doc), consider something like:

    min: {{tables.collections.readCapacityUnits.min || something.autoscaling.readCapacityUnitDefaults.min}}

That way we can set sensible global defaults and adjust as needed.

Reply:
Obviously similar for all min/max fields and anything that gets repeated over and over with the same values in almost every location.